src/modules/coreinit/coreinit_exit.cpp

```diff
@@ -0,0 +1,24 @@
+#include "coreinit.h"
+#include "coreinit_exit.h"
+#include "utils/log.h"
+
+void
+_Exit()
```

Comment:
True, but this follows your existing pattern of library implementation function names. Also, since this function uses C++ linkage, its true name will be mangled and thus won't collide with any standard libraries.